### PR TITLE
Create examples

### PR DESCRIPTION
**Pull request by shannonhaughton**
Adding examples to exported functions

### CODE REVIEW

**Review by statasaurus — APPROVED** [review]
🐐 Thank you for adding in all these examples!

File: R/tfrmt_sigdig.R [hunk 1]
```diff
@@ -168,9 +168,12 @@ param_set <- function(...){
 #'           values = value,
 #'           sorting_cols = vars(ord1, ord2, ord3),
 #'           col_plan = col_plan(-starts_with("ord"))) %>%
-#'     print_to_gt(data_labs)
+#'     print_to_gt(labs_data)
```
Comment (statasaurus) [inline R/tfrmt_sigdig.R:171]: I just realized some of these need to be changed to reflect the new data names

**Review by statasaurus — CHANGES_REQUESTED** [review]
You just need to go through and change the datasets from *_data to data_*

**Comment by shannonhaughton** [comment]
@statasaurus I think there was only one case of this, have done a find in files and cant see any more

**Review by statasaurus — APPROVED** [review]
Amazing 🐄

**Pull request merged**